frontend/src/app/app.component.css

```diff
@@ -0,0 +1 @@
+
```

Comment:
Delete this file (major sonar codesmell).

Reply:
Ich habe einfach die leere Zeile gelöscht. Die Datei könnte ganz praktisch sein für die zwei andere Teammitglieder weil es ja eine Standard Datei ist (und die sich dann nicht wundern wo die ist). Wenn Sonar weiter meckert dann nehme ich sie raus